mapie/control_risk/risks.py

```diff
@@ -0,0 +1,130 @@
+from typing import cast
+
+import numpy as np
+from numpy.typing import NDArray
+from sklearn.utils.validation import column_or_1d
+
+
+def _compute_recall(
+    lambdas: NDArray,
+    y_pred_proba: NDArray,
+    y: NDArray
+) -> NDArray:
+    """
+    Compute the risks (recall)
```

Comment:
UPD: improve the description of this risk.

Reply:
Done